tests/unit-tests/dds/DCPS/PrinterValueWriter.cpp

```diff
@@ -401,8 +401,8 @@ TEST(dds_DCPS_PrinterValueWriter, write_float128)
 TEST(dds_DCPS_PrinterValueWriter, write_fixed)
 {
   PrinterValueWriter pvw;
-  pvw.write_fixed(OpenDDS::FaceTypes::Fixed());
-  EXPECT_STREQ(pvw.str().c_str(), "fixed");
+  pvw.write_fixed(ACE_CDR::Fixed::from_string("1"));
```

Comment:
There's a ubsan error from a call to `ACE_CDR::Fixed::from_string` but looks like it's an ACE issue.

Reply:
Yes, I'm looking at that now. Thanks